### PR TITLE
Update: eslint --env-info output os info

### PR DESCRIPTION
**Pull request by aladdin-add**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/master/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ x] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
`eslint --env-info` print the os info, it looks like:
```bash
Environment Info:

Node version: v15.5.0
npm version: v7.3.0
Local ESLint version: v7.19.0 (Currently used)
Global ESLint version: Not found
Operating System: darwin 20.3.0
```

#### Is there anything you'd like reviewers to focus on?
no.

### CODE REVIEW

**Comment by nzakas** [comment]
I’m not sure it’s worth adding another dependency just to output debug information. Couldn’t we just use process.platform?

**Comment by aladdin-add** [comment]
the package can output more readable info, e.g.
```js
osName('win32', '6.3.9600');
//=> 'Windows 8.1'
```

I think either is fine. will remove the dependency if you prefer that way. :)

**Comment by nzakas** [comment]
As I said, I just don’t think this is important enough to add another dependency. Win32 or Win64 is plenty of info for our purposes.

**Comment by aladdin-add** [comment]
just rebased the latest master.

**Review by nzakas — APPROVED** [review]
LGTM

**Comment by mdjermanovic** [comment]
I think we can already mark this as accepted, since it matches our new bug report template.

File: tests/lib/shared/runtime-info.js [hunk 1]
```diff
@@ -57,6 +58,8 @@ describe("RuntimeInfo", () => {
         let spawnSyncStubArgs;
 
         beforeEach(() => {
+            os.platform = () => "darwin";
+            os.release = () => "20.3.0";
```
Comment (mdjermanovic) [inline tests/lib/shared/runtime-info.js:62]: Can we use `sinon` here to prevent this change from leaking to other tests?

**Review by mdjermanovic — CHANGES_REQUESTED** [review]

**Review by nzakas — APPROVED** [review]

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**